com.unity.ml-agents/Runtime/Sensors/ObservationWriter.cs

```diff
@@ -144,6 +145,27 @@ public void AddRange(IEnumerable<float> data, int writeOffset = 0)
             }
         }
 
+        public void AddList(IList<float> data, int writeOffset = 0)
```

Comment:
Same comments for overloading resolution, but I made this unambiguous (and marked the old version obsolete).